package.json

```diff
@@ -71,7 +71,7 @@
     "lint:css": "stylelint './packages/**/src/**/*.tsx'",
     "postinstall": "polkadot-dev-yarn-only",
     "postinstall:electron": "electron-builder install-app-deps",
-    "test": "polkadot-dev-run-test packages/page-claims/src packages/apps-electron",
+    "test": "polkadot-dev-run-test packages/page-claims/src packages/apps-electron packages/page-accounts",
```

Comment:
This is the only reason why not merging as-is since this will do these tests on all runs. Would suggest moving this into a seperate test call (which we can then do with a specific GH workflow).